CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.13.0 - 2025-08-07
+- in a massive coincidence, python 3.13 support in version 3.13.0
```

Comment:
😆 
👇

Reply:
Once I realized I wanted to do a minor bump for 3.12 anyway I had to do it